tests/test_job_management.py

```diff
@@ -124,6 +132,7 @@ def test_email_callback_after_run_analysis(dispatcher_live_fixture, dispatcher_l
         # makes more sense to have it sent directly with the request,
         # though it is not considered for the url encoding
         # to confirm
+        # VS: I do not really understand, why is it useful in the request? can dispatcher not compute it? Relying on time from another locations may cause issues
```

Comment:
Do you mean to have it acquired on the dispatcher's side?

It indeed makes sense, it will help also in avoinding issues related to requests coming from different time zones?

Reply:
exactly! timezones, time sync, client errors and manipulations, etc